README.md

```diff
@@ -30,28 +30,19 @@ In this repo the fuzzers have the following structure:
 
 ## User Guide
 
-### 1. create the docker container 
-* Use the command  `docker build -t ready-to-fuzz-image .` inside this project to create a docker image named **ready-to-fuzz-image** from the dockerfile in the project.
+### 1. Create the docker container 
+To create and run the docker container ready to run the fuzzers use the command:
```

Comment:
This phrase is confusing. I think it should be

```suggestion
To create and run the docker container with the fuzzers use the command:
```

Reply:
done 👍